packages/ui/src/components/molecules/CardHome.tsx

```diff
@@ -16,8 +16,11 @@ const CardContainer = styled(FlexBox)`
   padding: ${dimensions.spacing.md};
 `
 
-type CardHomeProp = Pick<TCardHome, 'backgroundImg'>
+// type CardHomeProp = Pick<TCardHome, 'backgroundImg'>
 
+type CardHomeProp = {
+  backgroundimg?: string
+}

```

Comment:
I think there is no need to change the name to lower case. You could use shouldForwardProp from styled components https://styled-components.com/docs/api#using-custom-props